flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -427,6 +427,44 @@ protected static Notification createNotification(
     setProgress(notificationDetails, builder);
     setCategory(notificationDetails, builder);
     setTimeoutAfter(notificationDetails, builder);
+
+    if (notificationDetails.bubbleActivity != null) {
+      try {
+        Class cls = Class.forName(notificationDetails.bubbleActivity);
+        Intent testIntent = new Intent(context, cls);
+        testIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+
+        if (notificationDetails.bubbleExtra != null) {
+          final Bundle extra = new Bundle();
+          extra.putString("bubbleExtra", notificationDetails.bubbleExtra);
+          testIntent.putExtras(extra);
+        }
+
+        int actionFlags = PendingIntent.FLAG_MUTABLE | PendingIntent.FLAG_UPDATE_CURRENT;
+        PendingIntent bubbleIntent =
+            PendingIntent.getActivity(context, notificationDetails.id, testIntent, actionFlags);
+
+        IconCompat icon =
+            IconCompat.createWithResource(
+                context, getDrawableResourceId(context, notificationDetails.icon));
+
+        Log.e(TAG, "Created pending intent: $bubbleIntent");
+        Log.e(TAG, bubbleIntent.toString());
+
+        if (!StringUtils.isNullOrEmpty(notificationDetails.shortcutId)) {
+
+          androidx.core.app.NotificationCompat.BubbleMetadata bubbleData =
+              new androidx.core.app.NotificationCompat.BubbleMetadata.Builder(bubbleIntent, icon)
+                  .setDesiredHeight(600)
+                  .build();
+
+          builder.setBubbleMetadata(bubbleData);
+        }
+      } catch (ClassNotFoundException e) {
+        e.printStackTrace();
```

Comment:
is there a reason for swallowing the exception? can this be surfaced as an exception on the Flutter with appropriate details for users of the plugin to handle?

Reply:
I'm not sure the best way to do this, I would appreciate some guidance here